tests/requirements-apt.txt

```diff
@@ -0,0 +1,4 @@
+libyaml-dev
```

Comment:
do we really need this indirection via file?
at least rename it to `apt.txt` it has nothing to do with python requirements.txt

Reply:
This is pretty standard in our other repositories.